coral.py

```diff
@@ -291,32 +305,42 @@ def config_prompt():
                     else:
                         n -= 1
                 elif event.key == pygame.K_RIGHT: 
-                    if configs[n] == 2:
-                        configs[n] = 0
+                    opcao = 'velocidade'
+                    match n:
+                        case 0:
+                            opcao = 'velocidade'
+                        case 1:
+                            opcao = 'tamanho'
+                        case 2:
+                            opcao = 'frequencia'
+                        case 3:
+                            opcao = 'volume'
+                    if configs[opcao] == 2:
+                        configs[opcao] = 0
                     else:
-                        configs[n] += 1
+                        configs[opcao] += 1
                     if n == 3:
                         update_volume()
                 elif event.key == pygame.K_LEFT:  
-                    if configs[n] == 0:
-                        configs[n] = 2
+                    if configs[opcao] == 0:
```

Comment:
The game crashes (error in line 325) when you open the game and try to change a setting using the left arrow. My guess is that 'opcao' is not defined in that scope.

Reply:
Should be working now! thanks for pointing this out
Don't know how it didn't cross my mind testing the left arrow key lol

Reply:
Nice! It's working now